_includes/releases/v23.1/v23.1.0-rc.1.md

```diff
@@ -0,0 +1,78 @@
+
+- Two views have been added to the [`crdb_internal` system catalog](../v23.1/crdb-internal.html): `crdb_internal.statement_activity` surfaces data in the persisted `system.statement_activity` table, and `crdb_internal.transaction_activity` surfaces the `system.transaction_activity` table. [#102047][#102047]
+- Span statistics are now unavailable on mixed-version clusters [#101878][#101878]
+
+<h3 id="v23-1-0-rc-1-db-console-changes">DB Console changes</h3>
+
+- Timestamps are now displayed in DB Console in the timezone specified in the `ui.display_timezone` [cluster setting](../v23.1/cluster-settings.html), if set. Currently supported timezones are `Coordinated Universal Time` and `America/New_York`. [#102241][#102241]
+
+<h3 id="v23-1-0-rc-1-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug introduced prior to v22.1 where an `EXPORT` statement could be incorrectly evaluated and result in a node panic or incorrect query results if it had projection or rendering on top of the `EXPORT`, such as the statement`WITH cte AS (EXPORT INTO CSV 'nodelocal://1/export1/' FROM SELECT * FROM t) SELECT filename FROM cte;`. Only the presentation of the query result was affected, not the exported data. [#101806][#101806]
+- The metric description of `rebalancing.readbytespersecond` and `rebalancing.writebytespersecond` now correctly reference bytes read and bytes written respectively. [#101710][#101710]
+- Fixed a rare bug introduced prior to v22.1 where distributed plans shutdown could cause the graceful drain of a node to become stuck retrying forever.This bug leads to errors like `drain details: distSQL execution flows:` with a non-zero number of flows that is not reducing over a long period of time. [#101884][#101884]
+- Fixed a bug where a `RESTORE` operation with `skip_localities_check` could fail with errors if regions were missing on a cluster. [#101798][#101798]
+- Fixed a bug introduced in testing releases of v23.1 that could cause incorrect results for queries with `STRICT` user-defined functions. [#101951][#101951]
+- Fixed a rare bug that can cause keys to be unexpectedly deleted locally within a store by replica rebalancing during a write heavy workload. [#102166][#102166]
+- Fixed a bug where a failed or cancelled `IMPORT` operation could leave some of the imported rows behind after it was cancelled, if the writing processes were slow enough to continue writing after the cleanup process started. [#102246][#102246]
+- Fixed a bug in the behavior of the `enforce_home_region` [session variable](../v23.1/set-vars) that may have allowed a hash join to be favored over a lookup join, or failed to error out remote accesses done by uniqueness checks for mutations on R`EGIONAL BY ROW` tables. Also, static erroring of some locality-optimized lookup joins is now handled dynamically during query execution. [#102287][#102287]
```

Comment:
- Link to RBR doc.
- The part that starts "Also, static erroring" looks like it should be split into a separate note with the same PR link.

```suggestion
- Fixed a bug in the behavior of the `enforce_home_region` [session variable](../v23.1/set-vars) that may have allowed a hash join to be favored over a lookup join, or failed to error out remote accesses done by uniqueness checks for mutations on `REGIONAL BY ROW` tables. Also, static erroring of some locality-optimized lookup joins is now handled dynamically during query execution. [#102287][#102287]
```